frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -851,7 +872,7 @@ function DataFrame({
           }}
           // Header click is used for column sorting:
           onHeaderClicked={(colIndex: number, _event) => {
-            if (isEmptyTable || isLargeTable || isColumnSelectionActivated) {
+            if (!isSortingEnabled || isColumnSelectionActivated) {
```

Comment:
now that we have a proper menu, do we want to keep the behavior that you cannot sort when selection is enabled? It might make sense to remove that restriction now because you cannot sort via a click on the column header, but by clicking on the column menu.

Reply:
> do we want to keep the behavior that you cannot sort when selection is enabled?  

With this PR, sorting columns via the menu will be possible when column selections are enabled. This logic here just disables sorting columns via click on the header since this aspect is incompatible with column selections (clicking on column needs to select the column).